plugins/lang-groovy/src/test/java/org/elasticsearch/script/groovy/GroovySecurityTests.java

```diff
@@ -107,6 +107,9 @@ public void testEvilGroovyScripts() throws Exception {
         // AccessControlException[access denied ("java.io.FilePermission" "<<ALL FILES>>" "execute")]
         assertFailure("def methodName = 'ex'; Runtime.\"${'get' + 'Runtime'}\"().\"${methodName}ec\"(\"touch /tmp/gotcha2\")");
 
+        // AccessControlException[access denied ("java.lang.RuntimePermission" "modifyThreadGroup")]
+        assertFailure("t = new Thread({ println 3 }); t.start(); t.join()");
```

Comment:
sure, i took the test from the old groovy sandboxing tests. but it will fail in the Thread ctor (which checks for access to modify the thread group) so we don't even need the start() either...